templates/wsl/index.html

```diff
@@ -60,12 +60,12 @@ <h3 class="p-heading--four">Access the Linux terminal on Windows</h3>
     <div class="col-4 p-divider__block">
       <h3 class="p-heading--four">Develop cross-platform</h3>
       <p>Build and debug Linux applications  with Windows tools like Visual Studio Code, Visual Studio, and JetBrains IDEs before deploying to the cloud.</p>
-      <p><a class="p-link--external" href="https://youtu.be/DmfuJzX6vJQ" target="_blank">Discover how to easily access "Kubernetes on Windows with WSL2" in our webinar</a></p>
+      <p><a class="p-link--external" href="https://youtu.be/DmfuJzX6vJQ" target="_blank" rel="noopener noreferrer">Discover how to easily access "Kubernetes on Windows with WSL2" in our webinar</a></p>
```

Comment:
This is simply a link to a video from a static page. If you want to retain people on the page embed the video